src/io.jl

```diff
@@ -11,6 +11,7 @@ Base.position(c::Cursor) = position(c.io)
 
 
 function unpack() end
+packedsizeof(T::Type) = sum(sizeof.(fieldtypes(T)))
```

Comment:
does `Base.summarysize()` give correct size here?

Reply:
No, that includes some padding, unfortunately.